server/app/mutations/grids/update.rb

```diff
@@ -8,9 +8,9 @@ class Update < Mutations::Command
 
     optional do
       hash :stats do
-        required do
+        optional do
           hash :statsd do
-            required do
+            optional do
```

Comment:
This can remain as `required`, the `stats` -> `optional` is enough to accept`{"stats":{"statsd":null}}`.